tests/nsjail/test_unix.py

```diff
@@ -80,3 +80,17 @@ def test_cannot_remount_rootfs(self):
         self.assertEqual(result.returncode, 1)
         self.assertEqual(result.stdout, "mount: only root can do that\n")
         self.assertEqual(result.stderr, None)
+
+    def test_fail_gracefully_when_linuxfs_not_setup(self):
+        orig_binary = self.nsjail.shell_binary
+        self.nsjail.shell_binary = "/nonexistent/path"
+        cmd = dedent("""
+            whoami
+        """)
+
+        result = self.nsjail.unix(cmd)
+        self.assertEqual(result.returncode, None)
+        self.assertEqual(result.stdout, "LinuxFS not set up")
+        self.assertEqual(result.stderr, None)
+
+        self.nsjail.shell_binary = orig_binary  # clean up
```

Comment:
Clean up shouldn't be necessary since the `setUp` fixture is executed separately for every test. In other words, the nsjail instance is re-created for each test rather than shared among them.